projects/core/src/event/event.service.ts

```diff
@@ -42,12 +48,14 @@ export class EventService {
    *  when the event source is no longer maintained by its creator
    * (i.e. in `ngOnDestroy` if the event source was registered in the component).
    *
+   * @since 3.1 - registers the given `source$` for the parent classes of the given `eventType`.
+   *
    * @param eventType the event type
    * @param source$ an observable that represents the source
    *
    * @returns a teardown function which unregisters the given event source
    */
-  register<T>(eventType: Type<T>, source$: Observable<T>): () => void {
+  register<T>(eventType: AbstractType<T>, source$: Observable<T>): () => void {
```

Comment:
This one should be changed to `Type` in the long run, so if not now because of some breaking changes, then we should leave some note/TODO here.

Reply:
Added TODO